ruby_programming/basic_ruby/project_advanced_building_blocks.md

```diff
@@ -312,7 +316,11 @@ You learned about the Enumerable module that gets mixed into the Array and Hash
 [Submit a link](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) to the github repo with your files in it here with a pull request.  Please include your partner's github handle somewhere in the description if they would like attribution.
 
 * Add your solution below this line!
+<<<<<<< HEAD
+* [Javier Machin's solution](https://github.com/Javier-Machin/Ruby_exercises/blob/master/my_methods.rb)
+=======
 * [Samuel Masters' Soultion](https://github.com/redeyetuning/rb_adv_bldg_blks/blob/master/enumerable_methods.rb)
+>>>>>>> 4951b07eec258113a037f9da01ebd616a1ae663b
```

Comment:
Please delete these merge lines and incorporate both yours and Samuel's solutions into this file